.github/workflows/ci.yml

```diff
@@ -11,48 +11,50 @@ jobs:
     strategy:
       matrix:
         dockerbase: ["centos7", "centos8"]
+        packagemanager: ["pip", "conda"]
       fail-fast: false
 
     steps:
     - name: Checkout repo
       uses: actions/checkout@v2
 
-    - name: Build docker images (base - ${{ matrix.dockerbase }})
+    - name: Build docker images (base - ${{ matrix.dockerbase }}:${{ matrix.packagemanager }})
       run: |
         docker build \
-          -t exaworks/sdk-base:${{ matrix.dockerbase }} \
-          --cache-from exaworks/sdk-base:${{ matrix.dockerbase }} \
+          -t exaworks/sdk-base_${{ matrix.packagemanager }}:${{ matrix.dockerbase }} \
```

Comment:
proposal as an initial setup for multiple input parameters:
- have an internal variable (e.g., `DOCKER_TAG`), which will incorporate all input parameters;
- use that internal variable for image tagging;

```
        strategy: ...
        env:
          DOCKER_TAG: "${{ matrix.dockerbase }}_${{ matrix.packagemanager }}"
```
and then just use `DOCKER_TAG`
```
          -t exaworks/sdk-base:${{ env.DOCKER_TAG }} \
```

p.s. and to have `DOCKER_TAG_LATEST: "centos7_pip"` for `deploy.yml`

Reply:
I like this a lot, thank you!